srcs/Channel.cpp

```diff
@@ -15,13 +15,13 @@ name(ch_name)
 	password = "";
 }
 
-// Needed for pair functionality
-Channel::Channel(const Channel &c): name(c.get_name())
-{
-	Client	*op = c.get_op();
-	this->clients.insert(std::make_pair<std::string, Client*>(op->getNickname(), op));
-	this->owner = op->getNickname();
-}
+// // Needed for pair functionality
```

Comment:
Why was it commented out?

Reply:
Ow, sorry, forgot about it. Commented it out because it is not implemented correctly, and system called it when it supposed to call other constructor, and because of that server didnt work correctly. We dont use it, but if we wanna leave it in the code then we have to implement it correctly. Also, we dont have copy constructors for other classes.

Reply:
Fair, seems the subject doesn't say to develop all the constructors